frontend/src/lib/components/account-page/SignIn.svelte

```diff
@@ -4,12 +4,22 @@
 	import Tx from 'sveltekit-translate/translate/tx.svelte';
 	import { getContext } from 'svelte';
 	import { CONTEXT_KEY, type SvelteTranslate } from 'sveltekit-translate/translate/translateStore';
+	import { M } from '$lib/stores/global';
 
 	const { t } = getContext<SvelteTranslate>(CONTEXT_KEY);
 
 	let isSignInButtonDisabled = false;
+
+	const authLink =
+		'https://github.com/sjerzykiewicz/NoName-Surveys/tree/dev?tab=readme-ov-file#-amu-usos-authentication';
+	const readmeLink =
+		'https://github.com/sjerzykiewicz/NoName-Surveys/tree/dev?tab=readme-ov-file#-noname-anonymous-surveys';
```

Comment:
this link will direct to 'dev' branch 
```suggestion
		'https://github.com/sjerzykiewicz/NoName-Surveys?tab=readme-ov-file#-noname-anonymous-surveys';
```